cmd/http/headers.go

```diff
@@ -156,6 +156,9 @@ const (
 	// Reports number of drives currently healing
 	MinIOHealingDrives = "x-minio-healing-drives"
 
+	// Object was stored with additional erasure codes due to degraded system at upload time
+	MinIOErasureUpgraded = "x-minio-erasure-upgraded"
```

Comment:
I think this needs to be `x-minio-internal-erasure-upgraded` ? @klauspost since `x-minio-` is used for external API communication.